learn/userguides/secondary-analysis/what-is-the-secondary-analysis-service/what-is-the-secondary-analysis-service.md

```diff
@@ -6,15 +6,16 @@ title: "What is the Secondary Analysis Service?"
 
 ## What is Secondary Analysis?
 
-Secondary analysis is the process by which a computational pipeline, designed to examine data generated by a specific assay, is used to analyze raw experimental data and distinguish valid data appropriate for further analysis from artifactual data resulting from biological or technical complications. In the HCA project, secondary analysis produces collections of quality metrics and features over some common space that can be used for further analysis. As an example, secondary analysis of single cell RNA-Seq data results in aligned, QC’d reads; a matrix of gene expression, and a matrix of quality control metrics describing the data.  
+Secondary analysis is the process by which a computational pipeline, designed to examine data from a specific assay, is used to analyze raw experimental data and distinguish valid, data appropriate for further analysis, from artifactual data resulting from biological or technical complications. In the HCA project, secondary analysis produces collections of quality metrics and features that can be used for further analysis. For example, secondary analysis of single cell RNA-Seq data results in aligned, QC’d reads; a matrix of gene expression, and a matrix of quality control metrics describing the data.   
 
 ## What is the Secondary Analysis Service?
-The secondary analysis service consists of analysis pipelines and execution infrastructure that move raw data through standardized pipelines, producing measurements which are then sent  back to the Data Storage Service for storage and download by anyone, including you! The HCA DCP ends up storing two types of data: the submitted raw data and data resulting from secondary analysis. Both types are available for download. As new single cell technologies and analysis methods are developed and accepted by the research community, we will implement new secondary analysis pipelines and make both the pipelines and the data publically available.  
+The secondary analysis service consists of analysis pipelines and execution infrastructure that move raw data through analysis, producing measurements that are sent  back to the Data Storage Service for storage and download by anyone, including you! The HCA DCP stores both the submitted raw data and data resulting from secondary analysis, and each type is available for download. As new single cell technologies and analysis methods are developed and accepted by the research community, we will implement new secondary analysis pipelines and make both the pipelines and the data publically available.  
```

Comment:
> The secondary analysis service consists of analysis pipelines and execution infrastructure that move raw data through analysis, producing measurements that are sent  back to the Data Storage Service for storage and download by anyone, including you! The HCA DCP stores both the submitted raw data and data resulting from secondary analysis, and each type is available for download. As new single cell technologies and analysis methods are developed and accepted by the research community, we will implement new secondary analysis pipelines and make both the pipelines and the data publically available. 

If Data Storage Service is capitalized, then Secondary Analysis Service should be, too.
Is there an extra space between **are sent** and **back**? 
Maybe it should say "are ingested into the Data Storage Service" instead? To imply that Ingest is required/used.